### PR TITLE
Added Github Events API Integration to BLACC-Smith

### PR DESCRIPTION
**Pull request by mikaylaorange**
If the user gives the bot a "Feature:" command with an issue title, it will automatically create an issue with a feature label to the blacc-smith repository. Each issue will be under Garrett's name in GitHub

### CODE REVIEW

**Review by DarrylBrooks97 — CHANGES_REQUESTED** [review]

**Review by Garrett1Tolbert — APPROVED** [review]

File: functions/githubRequests/githubRequests.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+const { blaccSmithChannel } = require('../constants');
+
+exports.handleNewIssue = async ({ channel, issue, access_token }) => {
+	try {
+		if (channel.id === blaccSmithChannel) {
+			return await postIssue({ issue, access_token });
+		} else
+			channel.send(
+				'All feature requests must be sent to the `#blacc-smith` channel'
+			);
```
[the remark this answers is not in the document]
Comment (Garrett1Tolbert) [reply functions/githubRequests/githubRequests.js:11]: Here he goes again 😂

[the remark this answers is not in the document]
Comment (DarrylBrooks97) [reply functions/githubRequests/githubRequests.js:11]: 🤪

[the remark this answers is not in the document]
Comment (mikaylaorange) [reply functions/githubRequests/githubRequests.js]: Done

**Review by DarrylBrooks97 — APPROVED** [review]
- [x] Creates issue from discord server

**Pull request merged**